ciongke/cmd/hook/main.go

```diff
@@ -59,12 +59,24 @@ var defaultJenkinsJobs = map[string][]JenkinsJob{
 			Context:   "Jenkins Federation GCE e2e",
 		},
 		{
+			Name:      "kubernetes-pull-build-test-gci-federation-e2e-gce",
+			Trigger:   regexp.MustCompile(`@k8s-bot federation (gce )?(e2e )?test this`),
```

Comment:
Although I don't know if I like the semantics of replacing `gce` with `gci` because technically the `gci` one is still running on `gce`. What about `(gci )?(gce )?(e2e )?` for the federation one and just `(gci )?(e2e )?` for the kubemark one, so that we are consistent in adding a `gci` "prefix" for triggering the gci ones?